src/Poseidon/Janno.hs

```diff
@@ -366,23 +417,24 @@ data JannoRow = JannoRow
     , jDateBCADStart                :: Maybe Int
     , jDateBCADStop                 :: Maybe Int
     , jDateType                     :: Maybe JannoDateType
+    , jDateNote                     :: Maybe String
     , jNrLibraries                  :: Maybe Int
-    , jDataType                     :: Maybe (JannoList JannoDataType)
+    , jCaptureType                  :: Maybe (JannoList JannoCaptureType)
     , jGenotypePloidy               :: Maybe JannoGenotypePloidy
     , jGroupName                    :: JannoStringList
```

Comment:
So I guess we could move these up. I think it wouldn't cause problems, since we don't match this gigantic data type by contractor anyway... right?

Reply:
Move up?

Reply:
Hmm, I guess I meant sex and group name